relayer/relays/parachain/config.go

```diff
@@ -5,20 +5,22 @@ import (
 	"fmt"
 
 	"github.com/snowfork/snowbridge/relayer/config"
+	beaconconf "github.com/snowfork/snowbridge/relayer/relays/beacon/config"
 )
 
 type Config struct {
-	Source   SourceConfig   `mapstructure:"source"`
-	Sink     SinkConfig     `mapstructure:"sink"`
-	Schedule ScheduleConfig `mapstructure:"schedule"`
+	Source        SourceConfig   `mapstructure:"source"`
+	Sink          SinkConfig     `mapstructure:"sink"`
+	Schedule      ScheduleConfig `mapstructure:"schedule"`
+	RewardAddress string         `mapstructure:"reward-address"`
 }
 
 type SourceConfig struct {
-	Polkadot  config.PolkadotConfig  `mapstructure:"polkadot"`
-	Parachain config.ParachainConfig `mapstructure:"parachain"`
-	Ethereum  config.EthereumConfig  `mapstructure:"ethereum"`
-	Contracts SourceContractsConfig  `mapstructure:"contracts"`
-	ChannelID ChannelID              `mapstructure:"channel-id"`
+	Polkadot  config.PolkadotConfig   `mapstructure:"polkadot"`
+	Parachain config.ParachainConfig  `mapstructure:"parachain"`
+	Ethereum  config.EthereumConfig   `mapstructure:"ethereum"`
+	Contracts SourceContractsConfig   `mapstructure:"contracts"`
+	Beacon    beaconconf.BeaconConfig `mapstructure:"beacon"`
```

Comment:
Why does the parachain relayer need the beacon config?

Reply:
https://github.com/Snowfork/snowbridge/pull/1321/files#r1827164159 for submit delivery proof back to BH.